README.md

```diff
@@ -3,6 +3,8 @@
 SonarLint is a free IDE extension that empowers you to fix coding issues before they exist. More than a linter, SonarLint highlights issues that may result in Bugs and Security Vulnerabilities as you create your code, offering clear remediation guidance and educational help so you can fix them before the code is committed.
 SonarLint in VS Code supports analysis of JS/TS, Python, PHP, Java, C, C++, Go, and IaC code locally in your IDE. Use with SonarQube & SonarCloud for optimal team performance, and additional benefits.
 
+Check the [SonarLint for VS Code documentation](https://docs.sonarsource.com/sonarlint/vs-code/) for a full list of installation instructions, requirements, and feature details.
```

Comment:
Maybe we could highlight `up-to-date` instead of `full list`? wdyt

Reply:
Great idea!